lib/reducers/create-otp-reducer.js

```diff
@@ -74,6 +74,8 @@ export function getInitialState (userDefinedConfig, initialQuery) {
     routingTypes: [],
     stopViewer: {
       numberOfDepartures: 3, // per pattern
+      // Hide block ids unless explicitly enabld in config.
```

Comment:
typo: enabld

Reply:
Oh, the spellchecker did not catch that @evansiroky!?